qiskit/pulse/parameter_manager.py

```diff
@@ -387,3 +393,38 @@ def update_parameter_table(self, new_node: Any):
         visitor = ParameterGetter()
         visitor.visit(new_node)
         self._parameters |= visitor.parameters
+
+    def _unroll_param_dict(
+        self,
+        parameter_binds: Mapping[
+            Parameter | ParameterVector, ParameterValueType | list[ParameterValueType]
+        ],
+    ) -> Mapping[Parameter, ParameterValueType]:
+        """
+        Unroll parameter dictionary to a map from parameter to value.
+
+        Args:
+            parameter_binds: A dictionary from parameter to value or a list of values.
+
+        Returns:
+            A dictionary from parameter to value.
+        """
+        out = {}
+        for parameter, value in parameter_binds.items():
+            if isinstance(parameter, ParameterVector):
+                if not isinstance(value, (list, tuple)):
```

Comment:
Why does it check for `tuple` if the type hint only lists `list`? Perhaps everything should just be `collections.abc.Sequence`?

Reply:
The type is now typing.Sequence.